src/lib/ui/noise-sensor/NoiseSensorEventItem.tsx

```diff
@@ -0,0 +1,76 @@
+import type { Event } from 'seamapi'
+
+import { formatDateTime } from 'lib/dates.js'
+import { ClockIcon } from 'lib/icons/Clock.js'
+
+interface NoiseSensorEventItemProps {
+  event: Event
+}
+
+export function NoiseSensorEventItem({
+  event,
+}: NoiseSensorEventItemProps): JSX.Element {
+  const { date, time } = formatDateTime(event.created_at)
+
+  return (
+    <div className='seam-noise-sensor-event-item'>
+      <div className='seam-noise-sensor-event-item-column-wrap'>
+        <ClockIcon />
+
+        <div className='seam-noise-sensor-event-item-datetime-wrap'>
+          <p className='seam-noise-sensor-event-item-date'>{date}</p>
+          <p className='seam-noise-sensor-event-item-time'>{time}</p>
+        </div>
+      </div>
+
+      <div className='seam-noise-sensor-event-item-context-wrap'>
+        <p className='seam-noise-sensor-event-item-context-label'>
+          {getContextLabel(event)}
+        </p>
+        {getContextSublabel(event) != null && (
```

Comment:
When would this be null?

Reply:
(Missed this earlier!) This is again from upstream event types. It's a fallback I wanted to get in so that, just in case those two properties are missing, the sub label UI is hidden and shows the label in the center.